code/model/modelread.cpp

```diff
@@ -4476,16 +4476,19 @@ void model_update_instance(int model_instance_num, int submodel_num, flagset<Shi
 {
 	auto pmi = model_get_instance(model_instance_num);
 	auto pm = model_get(pmi->model_num);
-	model_update_instance(pm, pmi, submodel_num, flags);
+	model_update_instance(pm, pmi, nullptr, submodel_num, flags);
 }
 
-void model_update_instance(polymodel *pm, polymodel_instance *pmi, int submodel_num, flagset<Ship::Subsystem_Flags>& flags)
+/*
+ * This function handles copying submodel instance information to other submodel instances as appropriate.  All callers should pass nullptr for the
+ * copy_from parameter.  This parameter is used for copying data to other LODs, and is only specified from within model_update_instance itself.
```

Comment:
If the parameter is purely internal then maybe it would be better to just have a local function here that has the parameter and a "public" one where the parameter is not present?

Reply:
I guess that would make sense.